tests/test_thi.py

```
@@ -0,0 +1,50 @@
+import numpy as np
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)